1/test/run

```diff
@@ -192,6 +191,16 @@ function run_s2i_tests() {
 # Tests.
 
 # Normal tests
+echo "Normal test: 64 bit"
+CONTAINER_ARGS="-u 10000:0"
```

Comment:
should set the JVM explicitly to 64bit, because if we (somehow) run this test on a machine w/ low memory it's going to go through the 32 bit path.